packages/client/src/components/trainerStatus/TrainerStatus.tsx

```diff
@@ -10,27 +12,41 @@ export const GET_DOCKER_STATE = gql`
 
 export default function TrainerStatus(): ReactElement {
   const options = { pollInterval: 5000 };
-  const { data, loading, error } = useQuery(GET_DOCKER_STATE, options);
+  const { data, loading, error } = useQuery<GetDockerState>(GET_DOCKER_STATE, options);
 
   if (loading) return <p>connecting to trainer</p>;
   if (error) return <p>cant connect to trainer</p>;
 
+  let statusMessage;
+  switch (data?.dockerState) {
+    case DockerState.NO_DOCKER:
+      statusMessage = "ERROR: docker wont respond.";
```

Comment:
This is urgency level 1/100, but should the capitals and period messages be consistent? Like all punctuation or no, and capital first letters.

Reply:
Honestly go crazy. As long as it looks nice. Basically everything about the UI right now is temporary